### PR TITLE
Update posts.html.twig

### PR DESCRIPTION
**Pull request by dudewithamood**
Changed so that only actual posts can be selected.

### CODE REVIEW

File: platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig [hunk 1]
```diff
@@ -16,11 +16,15 @@
...
+    {% if posts %}
         {% for id,post in posts %}
             {% set id = post.id %}
             {% set options = options|merge([{value: id, text: post.post_title}])  %}
         {% endfor %}
 
         {% set field = field|merge(field.selectize.Options|default({}))|merge({selectize: { Options: options }}) %}
+    {% endif %}
+
+    data-selectize="{{ (field.selectize is defined ? field.selectize|json_encode()|e('html_attr') : '') }}"
```
Comment (w00fz) [inline platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig]: I don't think this is needed? `posts` already extends the `selectize` field which adds the data-set on its own.

Comment (adi8i) [inline platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig]: Without data-set you can type in anything, in this case the input should allow you to input only available posts. Same as wordpress.categories.

Comment (w00fz) [inline platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig]: Then you probably just need to change line 25 and force `create` to false?

```
{% set field = field|merge(field.selectize.Options|default({}))|merge({selectize: { Options: options, create: false }}) %}
```

Comment (adi8i) [inline platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig]: Ok, you are the master here.

Comment (w00fz) [inline platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig]: That works?

Comment (adi8i) [inline platforms/wordpress/gantry5/admin/templates/forms/fields/wordpress/posts.html.twig]: Yup

**Pull request closed without merging**